roles/nginx/templates/kalite-nginx.conf

```diff
@@ -0,0 +1,59 @@
+# NGINX configuration for KA Lite
+#
+# Upstream KA-Lite server uses port 7007
+# Nginx proxy for KA-Lite uses port 8008
+#
+# If you want the website to be accessible at a different port, change
+#  PROXY_PORT = nnnn setting in /var/ka-lite/.kalite/settings.py
+# and change the below accordingly.
+
+
+upstream kalite {
+    server 127.0.0.1:7007;
```

Comment:
what sets the port used at 7007 when kalite_server_port: 8008 is in default_vars.yml?